src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory_Methods.cs

```diff
@@ -243,75 +243,68 @@ internal ImmutableArray<IArgumentOperation> DeriveArguments(BoundNode containing
 
         internal ImmutableArray<IArgumentOperation> DeriveArguments(BoundNode containingExpression)
         {
-            switch (containingExpression.Kind)
```

Comment:
>switch (containingExpression.Kind) [](start = 12, length = 34)

Please undo the refactoring. I believe this big switch is easier to follow when it uses a statement form.

Reply:
Please keep changes to the necessary minimum to implement the feature. Addition of a ne case shouldn't trigger rewrite of all existing cases.

---
In reply to: [428807548](https://github.com/dotnet/roslyn/pull/44436#discussion_r428807548) [](ancestors = 428807548)